.github/workflows/ci.yml

```diff
@@ -3,31 +3,36 @@ name: CI
 on:
   push:
     branches:
-    - master
+      - master
```

Comment:
Revert _all_ the whitespace changes.

Reply:
Reverted (I used the files from base repo, where the whitespaces were like this.)